src/fsql/query.py

```diff
@@ -10,7 +10,7 @@
 from abc import ABC, abstractmethod
 from dataclasses import dataclass, field
 from enum import Enum, auto
-from typing import Callable, Optional
+from typing import Callable, Iterable, Optional
```

Comment:
Shouldn't we rather use `from collections.abc import Iterable` (PEP585)?

Reply:
done